easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.6-GCC-8.3.0.eb

```diff
@@ -0,0 +1,34 @@
+name = 'OpenBLAS'
+version = '0.3.6'
```

Comment:
OpenBLAS 0.3.7 is out now. May be better to use that, as it eliminates the avx512 patch.